app/views/course/assessment/question/scribing/_scribing.html.slim

```diff
@@ -0,0 +1,8 @@
+- edit_path = edit_course_assessment_question_scribing_path(current_course, @assessment,
+  scribing)
+- delete_path = course_assessment_question_scribing_path(current_course, @assessment,
+  scribing)
+= render partial: 'course/assessment/questions/question',
+  locals: { question: scribing,
+    edit_path: edit_path,
```

Comment:
Group all the APIs related to scribing questions in `client/app/api`, these paths should be handled there ( instead of passing from backend ).